docs/source/intervention_models/lipid_based_nutrient_supplements/index.rst

```diff
@@ -0,0 +1,267 @@
+.. role:: underline
+    :class: underline
+..
+  Section title decorators for this document:
+
+  ==============
+  Document Title
+  ==============
+
+  Section Level 1
+  ---------------
+
+  Section Level 2
+  +++++++++++++++
+
+  Section Level 3
+  ~~~~~~~~~~~~~~~
+
+  Section Level 4
+  ^^^^^^^^^^^^^^^
+
+  Section Level 5
+  '''''''''''''''
+
+  The depth of each section level is determined by the order in which each
+  decorator is encountered below. If you need an even deeper section level, just
+  choose a new decorator symbol from the list here:
+  https://docutils.sourceforge.io/docs/ref/rst/restructuredtext.html#sections
+  And then add it to the list of decorators above.
+
+.. _lipid_based_nutrient_supplements:
+
+========================================================
+Small quantity lipid based nutrient supplements (SQ-LNS)
+========================================================
+
+Ideally, infants are breastfed for two years or longer, with complementary food introduced at six months of age. Diets of infants and young children aged six to 23 months need to include a variety of nutrient-dense foods, preferably from local sources, to ensure their nutrient needs are met. However, children's diets are likely to be deficient in macronutrients and micronutrients, specifically essential fatty acids, when nutrient-rich diets are not available to them in resource-poor settings. Various interventions are recommended, or have been used, to improve child malnutrition. This document focuses on small quantity lipid based nutrient supplements (SQ-LNS) as an intervention to improve malnutrition, particularly child wasting and stunting. Particularly, it draws on the 2019 Cochrane Review by Das et. al. [DAS_Cochrane_Review_2019]_
+
+.. todo::
+
+  Add a brief introductory paragraph for this document.
+
+.. contents::
+   :local:
+   :depth: 1
+
+.. list-table:: Abbreviations
+  :widths: 15 15 15
+  :header-rows: 1
+
+  * - Abbreviation
+    - Definition
+    - Note
+  * - LNS
+    - lipid-based nutrient supplements
+    - 
+  * - SQ
+    - small quantity
+    - 
+  * - FBF 
+    - fortified blended foods
+    -
+  * - CSB ++
+    - corn soy blend plus
+    -
+  * - MNP
+    - Multiple micronutrient powders
+    - 
+
+.. todo::
+
+  Fill out table with any abbreviations and their definitions used in this document.
+
+Intervention Overview
+-----------------------
+
+:underline:`Lipid-based nutrient supplements`
+
+Supplementary feeding is a strategy that includes provision of extra food to children beyond the normal ration of their home diets
+and is aimed at improving the nutritional status or preventing the nutritional deterioration of the target population. One of the nutritional
+interventions advocated to address malnutrition among children is **lipid-based nutrient supplements (LNS)**. LNS are a family of
+products designed to deliver nutrients to vulnerable people. They are considered 'lipid-based' because most of the energy provided
```

Comment:
Is this background information all from the das cochrane review too? If not it might be good to cite a review that others could read for more background info!

Reply:
hehe Yes- all this info I took from the Cochrane review. Of course in the review they had more specific references. I will add those references as I refine :)